.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "helm-charts/sumologic-kubernetes-collection"]
```

Comment:
I still believe we should rethink this and maybe go for `git subtree`

> Submodules are easier to push but harder to pull – This is because they are pointers to the original repository
> Subtrees are easier to pull but harder to push – This is because they are copies of the original repository

Taking this as an advice I believe we're more likely to pull our collection's chart rather than push to it from this repo.

https://martowen.com/2016/05/01/git-submodules-vs-git-subtrees/

Reply:
Not sure about that, after reading summary

Reply:
I'll rethink after confirmation that I don't need to make any changes in Helm chart and subcharts.